lib/source.re

```diff
@@ -25,8 +25,16 @@ let to_local_path: t => local =
       tempdir;
     };
 
+let is_local_template_dir = s => {
+  Utils.Filename.(
+    test(Exists, s)
+    && test(Exists, s ++ "/spin")
```

Comment:
Here you could use [`Utils.Filename.join`](https://github.com/tmattio/spin/blob/master/lib/utils.re#L140). We don't support Windows currently, but if we can keep it in mind when adding code, it will make our job easier when we work on supporting it.

Reply:
Much better, I guess I didn't properly look at `Utils.Filename` 🙄 Will update now.